datahub-web-react/src/app/entity/shared/tabs/Dataset/Schema/useGetEntitySchema.tsx

```diff
@@ -0,0 +1,24 @@
+import { cloneDeep } from 'lodash';
+import { EntityType } from '../../../../../../types.generated';
+import { useEntityData } from '../../../EntityContext';
+import { useGetDatasetSchemaQuery } from '../../../../../../graphql/dataset.generated';
+import { combineEntityDataWithSiblings, useIsSeparateSiblingsMode } from '../../../siblingUtils';
+
+export const useGetEntityWithSchema = () => {
+    const { urn, entityData, entityType } = useEntityData();
+    // Load the dataset schema lazily.
+    const { data: rawData, loading } = useGetDatasetSchemaQuery({
```

Comment:
also i think this query got lost in the commit so it's not defined? or is it already defined?

Reply:
it should be inside dataset.graphql!